NEWS.md

```diff
@@ -10,3 +10,9 @@ CRUD interface for:
 
 ## 1.0.3 Release
 * resolved missing release artifacts
+
+## 2.0.0 Release
+* Provided additional API endpoints to manipulate sub-vendor model data
+* Flattened vendor record data models (ie. edi, edi_ftp, edi_job)
```

Comment:
If you can provide links to the JIRA stories / bugs that would be more consistent with the other repos.